server/index.js

```diff
@@ -1,5 +1,8 @@
 const express = require('express');
 const path = require("path");
+const db = require('../database/seeder.js');
+console.log(db);
```

Comment:
Take out console.log after confirming that you have the right file

Reply:
Done

Reply:
Tim, I've addressed your comments. Please review.